scripts/ssl-manager.js

```diff
@@ -868,6 +920,8 @@ function SSLManager(config) {
 
     //managing certificate challenge validation by routing all requests to master node with let's encrypt engine
     me.manageDnat = function manageDnat(action) {
+        if (config.webroot) return { result: 0 };
```

Comment:
manageDnat is visible in CS log in your case

Reply:
but cmd command in it wont

Reply:
so fix the log. You need to check webroot outside the manageDnat function